python/tvm/driver/tvmc/target.py

```diff
@@ -181,6 +181,12 @@ def validate_targets(parse_targets, additional_target_options=None):
     if additional_target_options is not None:
         for target_name in additional_target_options:
             if not any([target for target in parse_targets if target["name"] == target_name]):
+                # When built with USE_MRVL=ON, add-on target options are passed from MRVL codegen's
+                # config which has pass_default=True and compiled with default target, don't error
+                # Use case: --target="llvm" cnn.onnx
+                if (len(tvm_targets) == 1) and (target_name == "mrvl"):
```

Comment:
TVMC would allow you to use multi-backend in many cases, so this conditional here seems too specific on a generic function. Can you propose a more generic solutionf?

Reply:
Thanks Leandron, I've taken your feedback , I'll be revisiting with a generic fix